tests/Supportive/DependencyInjection/DeptracExtensionTest.php

```diff
@@ -307,10 +315,12 @@ public function testNullAnalyser(): void
             ],
         ];
 
-        $this->expectException(InvalidConfigurationException::class);
-        $this->expectExceptionMessage('The child config "types" under "deptrac.analyser" must be configured.');
-
         $this->extension->load($configs, $this->container);
+
```

Comment:
The semantics of the test has been changed here.

Reply:
Yes - the way that the "analyser" config worked so far was that if it was given, you had to also specify the "types" member. But now, there are more members possible, and each one has a default. What we'd want is to say "if analyser is set, it has to contain at least one field, it cannot be null or empty, even though all members have defaults". But I could not find a way to achieve this in the config schema definition.

Reply:
...so now it is allowed to provide an empty value for it, with the defaults applying for each member fields.

Reply:
OK, makes sense. In that case, let's rename the test name to reflect the new meaning.